### PR TITLE
NX-OS: Add nxos_acls model 

### PR DESCRIPTION
**Pull request by adharshsrivatsr**

### CODE REVIEW

File: models/nxos/acls/nxos_acls.yml [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+      'supported_by': 'network'
+    }
+NETWORK_OS: nxos
+RESOURCE: acls
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_acls
+  version_added: '2.10'
+  short_description: Manage named IP Access Control Lists on NX-OS platform
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]:
```suggestion
  short_description: Manage named IP ACLs on Cisco NX-OS platform
```

File: models/nxos/acls/nxos_acls.yml [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+  description: Manage named IP Access Control Lists on NX-OS platform
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]:
```suggestion
  description: Manage named IP Access Control Lists on the Cisco NX-OS platform.
```

File: models/nxos/acls/nxos_acls.yml [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+  author: Bradley Thornton (@cidrblock) & Adharsh Srivats Rangarajan (@adharshsrivatsr)
+  notes:
+    - Tested against NX-OS 7.3.(0)D1(1) on VIRL
+    - <documentation for sequence number>
+    - To parse configuration text, provide the output of show running-config | section access-list or a mocked up config
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]: not sure what this line is for?

File: models/nxos/acls/nxos_acls.yml [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+  options:
+    running_config:
+      description:
+        - Used to parse given commands into structured format, only in parsed state
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]:
```suggestion
        - Parse given commands into structured format. Required if I(state=parsed).
```

Comment (samccann) [reply models/nxos/acls/nxos_acls.yml]: please add periods to the end of all descriptions here.

File: models/nxos/acls/nxos_acls.yml [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+      type: str
+    config:
+      description: A list of ACLs
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]:
```suggestion
      description: A dictionary of ACL options.
```

File: models/nxos/acls/nxos_acls.yml [hunk 6]
```diff
@@ -0,0 +1,395 @@
...
+                              type: str  
+                    prefix:
+                      description: Source network prefix
+                      type: str
+                    wildcard_bits:
+                      description: Destination wildcard bits
+                      type: str
+
+                dscp:
+                  description: Match packets with given dscp value
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]:
```suggestion
                  description: Match packets with given DSCP value.
```

File: models/nxos/acls/nxos_acls.yml [hunk 7]
```diff
@@ -0,0 +1,395 @@
...
+                              description: Router discovery solicitations
+                              type: bool
+                            source_quench:
+                              description: Source quenches
+                              type: bool
+                            source_route_failed:
+                              description: Source route failed
+                              type: bool
+                            time_exceeded:
+                              description: All time exceededs
```
Comment (samccann) [inline models/nxos/acls/nxos_acls.yml]:
```suggestion
                              description: All time exceeded.
```

File: models/nxos/acls/nxos_acls.yml (hunk 3 above)
Comment (adharshsrivatsr) [reply models/nxos/acls/nxos_acls.yml]: It was to add a comment there. Filled it now

File: models/nxos/acls/nxos_acls.yml (hunk 4 above)
Comment (adharshsrivatsr) [reply models/nxos/acls/nxos_acls.yml]: I(state=parsed) --> what is this?

**Pull request merged**